src/Microsoft.ML.CpuMath/Properties/AssemblyInfo.cs

```diff
@@ -3,8 +3,11 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 using Microsoft.ML.Internal.CpuMath.Core;
 
+[assembly: DefaultDllImportSearchPaths(DllImportSearchPath.SafeDirectories)]
+
```

Comment:
This fixes CA5392 on the CpuMath project. Notice that there were several CA5392 errors on several places of CpuMath, but @sharwell adviced to better add this attribute in the `AssemblyInfo.cs` of the project, in order to automatically fix it for all the project. This pattern of adding the solution on `AssemblyInfo.cs` of a given project might be useful for solving other errors.

**Note:** @sharwell wasn't entirely sure if this is the _correct_ way of fixing CA5392 , so we need to confirm it is the correct solution.

Reply:
@dotpaul I believe that adding the `DefaultDllImportSearchPaths` attribute is correct, but I need confirmation that we have used the correct argument value.

Reply:
It seems several CI builds are failing when trying to import the CpuMath DLL, so I think that more investigation would be needed for this particular solution suggestion.

Reply:
Let's first verify what the argument for this attribute needs to be, and once that is completed we can update the build to ensure this dependency is in the expected location for the tests to complete.

Reply:
Where is CpuMath.dll expected to be (e.g. same directory as Microsoft.ML.CpuMath.dll or whatev)? Should use the DefaultDllImportSearchPaths that corresponds to that.